tests/test_structsvm.py

```diff
@@ -77,11 +80,106 @@ def test_structsvm():
             raise ValueError(f"Ground truth {gt} for edge ({u},{v}) not valid.")
 
 
-try:
-    QuadraticSolver(2, 0)
-except RuntimeError:
-    test_structsvm = pytest.mark.xfail(test_structsvm)
+def create_noise_solver(graph):
+    solver = motile.Solver(graph)
+
+    solver.add_constraints(MaxParents(1))
+    solver.add_constraints(MaxChildren(1))
+
+    solver.add_costs(NodeSelection(weight=1.0, attribute="score", constant=-10.0))
+    solver.add_costs(EdgeSelection(weight=10.0, attribute="score", constant=3.0))
+    solver.add_costs(Appear(constant=10.0))
+
+    solver.add_costs(
+        NodeSelection(
+            weight=np.random.randint(-10, 10, 1)[0],
+            constant=np.random.randint(-10, 10, 1)[0],
+            attribute="noise",
+        ),
+        name="NodeNoise",
+    )
+    solver.add_costs(
+        EdgeSelection(
+            weight=np.random.randint(-10, 10, 1)[0],
+            constant=np.random.randint(-10, 10, 1)[0],
+            attribute="noise",
+        ),
+        name="EdgeNoise",
+    )
+
+    print("====== Initial Weights ======")
```

Comment:
how would you feel about removing the print statements here?  They're certainly useful for debugging, but they make it rather hard to run general `pytest -s` and make sense of the output.  perhaps an example file would be a better place to put all the prints?

Reply:
Good point, I use `logging` in debug mode now: [70d0ea6](https://github.com/funkelab/motile/pull/23/commits/70d0ea68bef7a42c54dfe151f9bea865ff3e2c46)

Reply:
The ilp solver in the back still talks for `pytest -s`, this needs to be addressed in `ilpy` with some verbosity flag.